dpnp/dpnp_iface_histograms.py

```diff
@@ -945,6 +1098,13 @@ def histogramdd(sample, bins=10, range=None, weights=None, density=False):
         values being used for the corresponding dimension.
         None is equivalent to passing a tuple of D None values.
 
+        Default: ``None``
+    density : {None, bool}, optional
+        If ``False`` or ``None``, the default, returns the number of
+        samples in each bin.
+        If ``True``, returns the probability *density* function at the bin,
+        ``bin_count / sample_count / bin_volume``.
+
         Default: ``None``
     weights : {dpnp.ndarray, usm_ndarray}, optional
```

Comment:
Need to clarify behavior with default `None` value.
And also please wrap `True` and `False` in the text below with ``.